src/ceph-disk

```diff
@@ -1468,12 +1468,32 @@ def prepare_dev(
         except subprocess.CalledProcessError as e:
             raise Error(e)
 
+def check_journal_reqs(args):
+    _, allows_journal = command([
+        'ceph-osd', '--check-allows-journal',
+        '-i', '0',
```

Comment:
Should all these be hardcoded to 0 ?

I see elsewhere in the script there are calls to ceph-osd that say this argument is ignored...